libcodechecker/cmd/cmd_line_client.py

```diff
@@ -259,54 +262,198 @@ def handle_list_results(args):
 
 
 def handle_diff_results(args):
-    def printResult(getterFn, baseid, newid, suppr, output_format):
+    def getDiffResults(getterFn, baseid, newid, suppr):
         report_filter = [
             codeCheckerDBAccess.ttypes.ReportFilter(suppressed=suppr)]
         add_filter_conditions(report_filter[0], args.filter)
-
-        sort_type = None
+        sort_mode = []
+        sort_mode.append(codeCheckerDBAccess.ttypes.SortMode(
```

Comment:
Okay, I agree that this long line is very ugly here. Also, expression can be simplified. Let's do this:

1. Introduce a new import at the top: 

```diff
  import codeCheckerDBAccess
  import shared
  from Authentication import ttypes as AuthTypes
+ from codeCheckerDBAccess import ttypes
```

Then rewrite this statement so the qualified name is only `ttypes.SortMode` and such.

Empty list and append can be simplified into a single list constructor: `sort_mode = [ttypes.SortMode(...)]`.